manager/manager.go

```diff
@@ -184,6 +184,10 @@ func New(memoryCache *memory.InMemoryCache, sysfs sysfs.SysFs, maxHousekeepingIn
 		inHostNamespace = true
 	}
 
+	if ignoreMetricsSet == nil {
+		ignoreMetricsSet = ignoreMetrics.MetricSet
```

Comment:
Good idea - so move the flag defined in this file to cadvisor.go in the project root and pass it down from there?